src/platform/linker/cl_macros.h

```diff
@@ -35,7 +35,7 @@
 
 #define UNDEF_SYMB_TYPE 0x1
 #define EXPORTED_SYMB_TYPE 0x2
-#define MAX_SYMBOLS 1024
+#define MAX_SYMBOLS 1024*64
```

Comment:
yes, this is a defense programming way to prevent variable capture by the surrounding environment.